### DISCUSSION TITLE
Feature request: `branchSuffix`/`additionalBranchSuffix`

### DISCUSSION THREAD

**Discussion by haldunanil** [renovatebot/renovate#17660] [Suggest an Idea]
We currently have a setup that only causes E2E tests to be run when a branch name ends in `-e2e`. For all renovate PRs, however, we want to enable E2E testing to make sure everything looks ok before merging.

I see in the docs that there's currently `branchPrefix`/`additionalBranchPrefix` option to prepend to a branch, but not one that lets me append to the end. It seems like modifying `branchName` is discouraged so I wanted to check here if an alternative can be provided.

Thanks!

**Comment by Gabriel-Ladzaretti** [comment]
[branchTopic](https://docs.renovatebot.com/configuration-options/#branchtopic) might be an option as well, but I not sure if its any different than editing the `branchName` directly.
That said, appending static string to it shouldn't be a problem with newly created branches, existing ones would probably be deleted and the corelated PRs be closed. 

[Other Exposed config options](https://docs.renovatebot.com/templates/).

Edit: you would probably need to edit any autoclosed PR title, as these would be ignored otherwise and not recreated.

**Comment by rarkins** [comment]
Here is the current definition: https://github.com/renovatebot/renovate/blob/997f7f1a426ad31d45a65bada653d9b41b411049/lib/config/options/index.ts#L1607-L1613

Adding `branchSuffix` to this could be OK. Note that `branchTopic` itself can essentially have suffixes too: https://github.com/renovatebot/renovate/blob/997f7f1a426ad31d45a65bada653d9b41b411049/lib/config/options/index.ts#L1621-L1628

**Reply by haldunanil** [reply]
Happy to open a PR to add it to the branchName if you think that might be the preferred approach, lmk!

**Reply by rarkins** [reply]
Yes, I think adding branchSuffix to branchName would be ok

**Reply by haldunanil** [reply]
Great, will put up a PR soon
